packages/manager/modules/nutanix/src/dashboard/general-info/redeploy/translations/Messages_fr_FR.json

```diff
@@ -28,5 +28,17 @@
   "nutanix_redeploy_custom_configuration_nodes": "Nœuds",
   "nutanix_redeploy_custom_config_ip_unique_error": "Cette IP est déjà utilisée sur ce cluster",
   "nutanix_redeploy_custom_config_ip_not_in_cidr_subnet_error": "Les adresses ne sont pas sur le même réseau.",
-  "nutanix_redeploy_button": "Redéployer"
+  "nutanix_redeploy_custom_config_ip_format_error": "Le format de l'IP n'est pas correct",
+  "nutanix_redeploy_button": "Redéployer",
+  "nutanix_redeploy_prism_element_vip": "Prism Element Virtual IP (VIP)",
+  "nutanix_redeploy_gateway_CIDR": "Gateway CIDR",
+  "nutanix_redeploy_prism_central": "Prism Central :",
+  "nutanix_redeploy_prism_central_type": "Type",
+  "nutanix_redeploy_prism_central_vip": "VIP",
+  "nutanix_redeploy_prism_central_ip": "IPs",
+  "nutanix_redeploy_replication_factor": "Facteur de réplication",
+  "nutanix_redeploy_erasure_coding": "Fonction Erasure Coding",
+  "nutanix_redeploy_aos": "AOS Version",
+  "nutanix_redeploy_node_ahv_ip": "AHV IP",
+  "nutanix_redeploy_node_cvm_ip": "CVM IP"
```

Comment:
**question:** Following content needs to be translated? IMHO it looks like static content.